docs/updates/migrate-v3.md

```diff
@@ -171,7 +171,7 @@ The following processors have been removed:
 
 | UI5 Tooling v2              | UI5 Tooling v3              | Note |
 | --------------------------- | --------------------------- | ------------------------- |
-| createDebugFiles<br/>uglify | minify                      | The minify task is executed earlier, before the bundling process takes place. Any existing 'beforeTask' or 'afterTask' configuration of custom tasks might need to be adapted to cater for this change. |
+| createDebugFiles<br/>uglify | minify                      | The minify task is executed earlier, before the bundling process takes place. Any existing `beforeTask` or `afterTask` configuration of custom tasks might need to be adapted to cater for this change. <br>To adapt, you can use the `generateResourcesJson` for subscription before or after the last standard task. By default, `generateResourcesJson` is disabled, but you can still subscribe to it, ensuring your custom task executes in the correct order. |
```

Comment:
Is there only one custom task? Otherwise it should read "...ensuring your custom tasks execute in the correct order."

Reply:
No, no, those are as many as needed. Thanks!